tests/test_common.py

```diff
@@ -1,9 +1,9 @@
-import unittest
 import tempfile
-import utils
-import os
+import unittest
 from os import path
-from common import *
+from recipemd.common import *
```

Comment:
Please do not import the package using the module name, this makes it necessary to install the package for testing
modify sys.path so that the module can be resolved, see also http://docs.python-guide.org/en/latest/writing/structure/#test-suite